### PR TITLE
Add BP observation schema

### LINKED ISSUE #644: Introduce BP observation schema and its conversion to json
#### Proposed solution

The BP metadata schemas had some further additions recently. [Bp.observation.xsd](https://github.com/imi-bigpicture/metadata-schema/blob/main/src/BP.observation.xsd) has been added to the BP metadata schema listing and its inclusion in our backend was requested.

#### DoD (Definition of Done)

BP observations data can be processed from an xml file, converted to json and added to database.

#### Testing

Unit and integration

### PR DESCRIPTION
**Pull request by csc-jm**
### Description

`BP.observation.xsd` schema found [here](https://github.com/imi-bigpicture/metadata-schema/tree/main/src) is added to our schemas. This also adds a new JSON schema that corresponds to the XML schema and updates the parser to make the json objects cleaner.

### Related issues

Fixes #644 

### Type of change

- [x] New feature (non-breaking change which adds functionality)

### Changes Made

- added `BP.bpobservation.xsd` and `bp_bpobservation.json` schema
- added an example test file for observations
- updated unit tests, integration tests and wordlist

### Testing

- [x] Unit Tests
- [x] Integration Tests

### CODE REVIEW

File: tests/integration/test_object.py [hunk 1]
```diff
@@ -61,15 +67,17 @@ async def crud_works(schema, filename):
...
                 res = await resp.json()
                 expected_true = not any(d["accessionId"] == accession_id[0] for d in res["metadataObjects"])
                 assert expected_true, f"object {accession_id[0]} still exists"
 
         gather_items = []
         for schema, filename in test_fega_xml_files:
-            gather_items.append(crud_works(schema, filename))
+            gather_items.append(crud_works(schema, filename, submission_fega))
+        for schema, filename in test_bigpicture_xml_files:
+            gather_items.append(crud_works(schema, filename, submission_bigpicture))
```
Comment (csc-felipe) [inline tests/integration/test_object.py:80]: Good to have this

Comment (csc-jm) [reply tests/integration/test_object.py:80]: Yea it seems `test_bigpicture_xml_files` were not being used in anything despite `conf.py` having it

File: metadata_backend/conf/schemas.json [hunk 2]
```diff
@@ -124,5 +124,14 @@
             "url": "https://github.com/imi-bigpicture/metadata-schema/blob/v0.1.1/src/BP.sample.xsd"
```
Comment (blankdots) [inline metadata_backend/conf/schemas.json:124]: should we put this in 0.2.0 ?

File: metadata_backend/conf/schemas.json [hunk 3]
```diff
@@ -124,5 +124,14 @@
...
         },
         "provider": "BP"
+    },
+    "observation": {
```
Comment (blankdots) [inline metadata_backend/conf/schemas.json:128]:
```suggestion
    "bpobservation": {
```

let us keep all with the bp prefix just in case

**Review by blankdots — CHANGES_REQUESTED** [review]

File: metadata_backend/conf/schemas.json (hunk 3 above)
Comment (csc-jm) [reply metadata_backend/conf/schemas.json:128]: Ok, then `image` should probably be changed to `bpimage` as well?

Comment (blankdots) [reply metadata_backend/conf/schemas.json:128]: yes

**Review by blankdots — APPROVED** [review]
considering how many changes we make to `MetadataXMLConverter` at some point we might want to derive other classes from that to fit specifics such as `BPMetadataXMLConverter`

Other than that note LGTM

**Pull request merged**